packages/desktop-client/src/components/schedules/LinkSchedule.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useCallback, useRef, useState } from 'react';
+
+import { useSchedules } from 'loot-core/src/client/data-hooks/schedules';
+import { send } from 'loot-core/src/platform/client/fetch';
+import { type Query } from 'loot-core/src/shared/query';
+
+import { type BoundActions } from '../../hooks/useActions';
+import Modal from '../common/Modal';
+import Search from '../common/Search';
+import Text from '../common/Text';
+import View from '../common/View';
+import { type CommonModalProps } from '../Modals';
```

Comment:
This is the circular dependency I meant.

- `LinkSchedule` depends on `Modals`
- `Modals` depends on `LinkSchedule`

![image](https://github.com/actualbudget/actual/assets/886567/4c82bd4b-8ae8-44f0-b2a9-de338bc1e9f4)